lib/cog/response.rb

```diff
@@ -5,18 +5,23 @@ class Cog
   class Response
     LOG_LEVELS = [ :debug, :info, :warn, :err, :error ]
 
-    attr_accessor :template, :content
+    attr_accessor :template, :content, :aborted
 
     def []=(key, value)
       @content ||= {}
       @content[key] = value
     end
 
-    def send
-      return if content.nil?
+    def abort
+      @aborted = true
+    end
 
+    def send
+      write "COGCMD_ACTION: stop" unless @aborted.nil?
```

Comment:
Maybe `COGCMD_ACTION: abort` for consistency?

Also, are there other values for `COGCMD_ACTION`?